anime_downloader/commands/config.py

```diff
@@ -1,6 +1,30 @@
-import click
 import os
+import click
 from tabulate import tabulate
+
+
+def check_running():
+    import psutil
+
+    already_running = False
+    dict_pids = {
+        p.info["pid"]: [p.info["name"], p.info['cmdline']]
+        for p in psutil.process_iter(attrs=["pid", "name", 'cmdline'])
+    }
+
+    if os.getpid() in dict_pids:
+        del dict_pids[os.getpid()]
+
+    for key, value in dict_pids.items():
+        if bool(value[1]):
+            if ('anime config' in ' '.join(value[1]) or
+                    'anime.exe config' in ' '.join(value[1])
+                        ) and ('python' in ' '.join(value[1])):
+                already_running = True
+    return already_running
+
+if check_running():
+    raise Exception('Another instance of "anime config" is already running!')
```

Comment:
"is active" or "is running" I think makes more sense here

Reply:
indeed